pkg/cli/debug/debug.go

```diff
@@ -362,6 +364,9 @@ func (o *DebugOptions) RunDebug() error {
 	if len(ns) == 0 {
 		ns = o.Namespace
 	}
+	if len(o.ToNamespace) > 0 {
```

Comment:
How is this different from explicitly using `--namespace`?

Reply:
example is:

oc debug istag/cli:latest -n openshift —to-namespace=my-test-ns

For when you don’t have write access in the namespace that contains the thing you want to debug.